src/popup/router/pages/Dashboard.vue

```diff
@@ -0,0 +1,189 @@
+          :description="$t('dashboard.back-up-card.description')"
+          is-big-card
+        >
+          <template #icon>
+            <InfoIcon />
+          </template>
+          <Button
+            class="card-button"
+            :text="$t('dashboard.back-up-card.button')"
+            new-ui
+            inline
+            no-margin
+            :to="{ name: 'settings-security' }"
+            fill="red"
+          />
+        </Card>
+      </CardRow>
+      <CardRow>
+        <Card
+          :title="$t('dashboard.buy-card.title')"
+          :description="$t('dashboard.buy-card.description')"
+          is-big-card
+          :background="buyBackground"
+        >
+          <template #icon>
+            <InfoIcon />
+          </template>
+          <Button
+            class="card-button"
+            :text="$t('dashboard.buy-card.button')"
+            new-ui
+            inline
+            no-margin
+            to="#"
+            fill="red"
+          />
+        </Card>
+      </CardRow>
+      <CardRow>
+        <Card
+          :title="$t('dashboard.name-card.title')"
+          :description="$t('dashboard.name-card.description')"
+          is-big-card
+          :background="chainNameBackground"
+        >
+          <template #icon>
+            <InfoIcon />
+          </template>
+          <Button
+            class="card-button"
+            :text="$t('dashboard.name-card.button')"
+            new-ui
+            inline
+            no-margin
+            to="#"
+            fill="purple"
+          />
+        </Card>
+      </CardRow>
+    </Plate>
+  </div>
+</template>
+
+<script>
+import { mapState, mapGetters } from 'vuex';
+import Plate from '../components/Plate.vue';
+import AccountSwitcher from '../components/AccountSwitcher.vue';
+import Card from '../components/dashboard/Card.vue';
+import CardRow from '../components/dashboard/CardRow.vue';
+import Button from '../components/Button.vue';
+import ArrowReceiveIcon from '../../../icons/arrow-receive.svg?vue-component';
+import ArrowSendIcon from '../../../icons/arrow-send.svg?vue-component';
+import InfoIcon from '../../../icons/info.svg?vue-component';
+import buyBackground from '../../../image/dashboard/buy-ae.jpg';
+import chainNameBackground from '../../../image/dashboard/chain-name.jpg';
+
+export default {
+  name: 'Account',
+  components: {
+    CardRow,
+    Card,
+    Plate,
+    AccountSwitcher,
+    ArrowReceiveIcon,
+    ArrowSendIcon,
+    InfoIcon,
+    Button,
+  },
+  data: () => ({
+    buyBackground,
+    chainNameBackground,
+  }),
+  computed: {
+    ...mapState(['backedUpSeed', 'transactions']),
+    ...mapGetters(['getAccountPendingTransactions']),
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/typography';
+@use '../../../styles/mixins';
+
+.account {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  .plate .header {
```

Comment:
I suggest to avoid styling class names that does not exist in current component's scope.